### PR TITLE
!ponyrp command

### PR DESCRIPTION
**Pull request by chev2**
This is a command that teleports a player to the Treatment Room when they use it. Typing anything into chat while in the Treatment Room replaces it with "i like ponies"

### CODE REVIEW

**Comment by ghost** [comment]
i 200% agree with this lol

File: lua/chatcommands/ponyrp.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
+RegisterChatCommand({'ponyrp'}, function(ply, arg)
+	if !Safe(ply) and !ply:InVehicle() then
```
Comment (swampservers) [inline lua/chatcommands/ponyrp.lua]: check Alive()

File: lua/chatcommands/ponyrp.lua [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+				if v:GetName() == "treatmentdoor" then v:Fire("Close") end
+				if v:GetName() == "treatmentlever" then v:Fire("PressOut") end
+			end
+		end
+	end
+end, {global=false, throttle=true})
+
+hook.Add("PlayerSay", "TreatmentRoomChat", function(ply, text, team)
+	if ply:GetLocationName() == "Treatment Room" then
+		return "i like ponies"
```
Comment (swampservers) [inline lua/chatcommands/ponyrp.lua:16]: people stuck in here won't really be able to call someone to get them out. I guess that's fine

Comment (ghost) [reply lua/chatcommands/ponyrp.lua:16]: they should notice something's wrong if the trapped guy keep saying "i love ponies" in global chat lol

Comment (chev2) [reply lua/chatcommands/ponyrp.lua:16]: yeah, it was a relatively cheap way to get out as well. they will just have to undergo pony heaven

**Pull request merged**